Front-End/src/components/Course.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import logo from './pics/logo.png';
+
+const Course = (props) => {
+	console.log(props);
+	return (
+		<div className='course'>
+			<div>
+				<img src={logo}></img>
+			</div>
+			<div className='course-details'>
+				<h2>Android Devlopment</h2>
+				<h3>Android Devlopment Android Devlopment Android Devlopment</h3>
+				<h3>Android Devlopment Android Devlopment Android Devlopment</h3>
+				<h3>Android Devlopment Android Devlopment Android Devlopment</h3>
+				<h3>Android Devlopment</h3>
+				<h3>Android Devlopment</h3>
+				<h3>Android Devlopment</h3>
+				<h3>Android Devlopment</h3>
```

Comment:
Why are these hard coded?

Reply:
Just as a place holder to create the general layout of the component. 
Will be dynamic in the next PR.